k8s.yaml

```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       containers:
       - name: ui
-        image: bitnami/kubeless-ui:master
+        image: bitnami/kubeless-ui:latest
```

Comment:
ok so the root manifest refers to latest, but the manifest in the release refers to the version.

Can we make sure that the CI actually builds a latest tag for HEAD ?

Reply:
> Can we make sure that the CI actually builds a latest tag for HEAD ?
￼
It will be difficult to make this change as in all our images we use latest to point to the last release

Reply:
Sorry, I think the scripts can be updated to make HEAD build the latest tag. I need to make the changes in the CI script in the test-infra repo. Can we merge this until that is implemented?

Reply:
Done!